### PR TITLE
Update README with Canary info

### PR DESCRIPTION
**Pull request by nguyen-dows**
This PR updates the README with information about Windows Terminal Canary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,6 +118,24 @@ repository.
 
 ---
 
+## Installing Windows Terminal Canary
+Windows Terminal Canary is a canary build of Windows Terminal. This canary build ships nightly with the latest code from our `main` branch before they make it to Windows Terminal Preview. Windows Terminal Canary is also our least stable offering, so you may discover bugs before we have had a chance to find them.
```
Comment (DHowett) [inline README.md:122]:
```suggestion
Windows Terminal Canary is a nightly build of Windows Terminal. This build the latest code from our `main` branch, giving you an opportunity to try features before they make it to Windows Terminal Preview.

Windows Terminal Canary is our least stable offering, so you may discover bugs before we have had a chance to find them.
```

File: README.md [hunk 2]
```diff
@@ -118,6 +118,24 @@ repository.
...
+
+Windows Terminal Canary is available as a App Installer distribution and a Portable ZIP distribution.
```
Comment (DHowett) [inline README.md:124]:
```suggestion
Windows Terminal Canary is available as an App Installer distribution and a Portable ZIP distribution. 
```

File: README.md [hunk 3]
```diff
@@ -118,6 +118,24 @@ repository.
...
 ---
 
+## Installing Windows Terminal Canary
+Windows Terminal Canary is a canary build of Windows Terminal. This canary build ships nightly with the latest code from our `main` branch before they make it to Windows Terminal Preview. Windows Terminal Canary is also our least stable offering, so you may discover bugs before we have had a chance to find them.
+
+Windows Terminal Canary is available as a App Installer distribution and a Portable ZIP distribution. 
+
+The App Installer distribution supports automatic updates. Due to platform limitations, this installer only works on Windows 11. 
+
+The Portable ZIP distribution is a portable application. It will not automatically update and or automatically check for updates. This portable ZIP distribution works on Windows 10 (19041+) and Windows 11.
```
Comment (DHowett) [inline README.md:130]: I'd add the "For more information . . . (link to distributions doc)" line under these if you can

File: README.md [hunk 4]
```diff
@@ -118,6 +118,26 @@ repository.
 
 ---
 
+## Installing Windows Terminal Canary
+Windows Terminal Canary is a nightly build of Windows Terminal. This build the latest code from our `main` branch, giving you an opportunity to try features before they make it to Windows Terminal Preview.
```
Comment (DHowett) [inline README.md:122]:
```suggestion
Windows Terminal Canary is a nightly build of Windows Terminal. This build has the latest code from our `main` branch, giving you an opportunity to try features before they make it to Windows Terminal Preview.
```

Comment (DHowett) [reply README.md:122]: sorry, i messed that one up

**Review by DHowett — APPROVED** [review]

**Review by DHowett — APPROVED** [review]

**Review by zadjii-msft — APPROVED** [review]

**Pull request merged**